Bugsnag/Client/BugsnagClient.m

```diff
@@ -163,11 +161,7 @@ void BSSerializeDataCrashHandler(const BSG_KSCrashReportWriter *writer, int type
         if (bsg_g_bugsnag_data.stateJSON) {
             writer->addJSONElement(writer, "state", bsg_g_bugsnag_data.stateJSON);
         }
-        if (bsg_g_bugsnag_data.breadcrumbsPath) {
-            // FIXME: This needs to be updated to cater for new breadcrumb storage scheme
-            // FIXME: addJSONFileElement generates broken JSON if the file cannot be opened
-            // writer->addJSONFileElement(writer, "breadcrumbs", bsg_g_bugsnag_data.breadcrumbsPath);
-        }
+        BugsnagBreadcrumbsWriteCrashReport(writer);
```

Comment:
This is more of a nit since I don't see the case where it would happen, but `BugsnagBreadcrumbsWriteCrashReport` takes a nonnull writer parameter, while `BSSerializeDataCrashHandler`'s writer is not marked as nonnull. I'd only see this becoming an issue if/when analyzer results are added to the test builds because it will probably set it off.

Reply:
`BSSerializeDataCrashHandler` does not defined any nullability and therefore defaults to `_Null_unspecified`.

This means the static analyser doesn't perform the checks - I have verified that it doesn't report any issues here

Reply:
Ah, good! I thought it might resolve to nullable 🙏